charmcraft/main.py

```diff
@@ -86,7 +86,7 @@ def run(self, parsed_args, all_commands):
         if parsed_args.command_to_help not in all_commands:
             # asked help on a command that doesn't exist
             msg = "no such command {!r}".format(parsed_args.command_to_help)
-            help_text = help_builder.get_usage_message("charmcraft", msg)
+            help_text = help_builder.get_usage_message(extra_command="", error_message=msg)
```

Comment:
`extra_command` could be an optional argument defaulting to "", unless you want to be very explicit about it.

Reply:
Ah, yes, I just talked about this in the other comment. I'll do it.